### PR TITLE
Pin IPython version in docs build to avoid RTD warning

### PR DESCRIPTION
**Pull request by sharadmv**

### CODE REVIEW

**Comment by jakevdp** [comment]
Oh, for Python 3.7 we'll need `ipython<=8.6.0`

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 absl-py
+ipython==8.6.0
```
Comment (jakevdp) [inline docs/requirements.txt:2]: Also add a TODO to fix ipython3 issue and remove this pinning

Comment (sharadmv) [reply docs/requirements.txt]: Done.

**Review by jakevdp — APPROVED** [review]
Thanks! Let's wait for the readthedocs build to finish before merging

**Pull request merged**